test/cpp/c10d/HashStoreTest.cpp

```diff
@@ -60,16 +60,16 @@ void stressTestStore(std::string prefix = "") {
   std::vector<std::thread> threads;
   c10d::test::Semaphore sem1, sem2;
   auto hashStore = c10::make_intrusive<c10d::HashStore>();
-  c10d::PrefixStore store(prefix, hashStore);
+  c10d::PrefixStore store(std::move(prefix), hashStore);
 
   for (C10_UNUSED const auto i : c10::irange(numThreads)) {
-    threads.emplace_back(std::thread([&] {
```

Comment:
Is this really any cleaner / more efficient without std::thread?

Reply:
It's a concurrency test